tests/integration/ovs_test.py

```diff
@@ -17,20 +17,33 @@
 # along with this program. If not, see <https://www.gnu.org/licenses/>.
 #
 
-import pytest
+from contextlib import contextmanager
+
 
+import libnmstate
 from libnmstate.schema import Interface
 from libnmstate.schema import InterfaceIPv4
+from libnmstate.schema import InterfaceState
+from libnmstate.schema import InterfaceType
 from libnmstate.schema import OVSBridge
 from libnmstate.error import NmstateLibnmError
 
+import pytest
```

Comment:
nit: This is a 3rd party lib, please place it after the std libs (after line 21).

Reply:
Done!